app/src/main/java/com/thinqtv/thinqtv_android/MainActivity.java

```diff
@@ -32,10 +37,19 @@ protected void onCreate(Bundle savedInstanceState) {
                 .setWelcomePageEnabled(false)
                 .build();
         JitsiMeet.setDefaultConferenceOptions(defaultOptions);
+
+        // COMMENTED OUT BECAUSE I'M NOT SURE IF IT ACTUALLY LOOKS GOOD. GET TEAM OPINION
+        //mainViewer = (VideoView) findViewById(R.id.videoview);
+        //Uri uri = Uri.parse("android.resource://" + getPackageName() + "/" + R.raw.crowdpubcommercialv_1);
+        //mainViewer.setVideoURI(uri);
+        //mainViewer.start();
```

Comment:
We probably shouldn't commit commented code like this to master. Instead make a branch where this code is uncommented and anyone can take a look at it, then remove it entirely here.

Reply:
Also, when removing code, make sure you remove any unused imports.